site.properties

```diff
@@ -89,29 +89,35 @@ ldap.lastNameAttribute  = sn
 
 # Database and JPA (H2)
 
-#jdbc.driver   = org.h2.Driver
-#jdbc.url      = jdbc:h2:../Manager-Repository;DB_CLOSE_DELAY=-1
-#jdbc.username = apromore
-#jdbc.password = MAcri
-#
-#jpa.database         = H2
-#jpa.databasePlatform = org.eclipse.persistence.platform.database.H2Platform
-#jpa.showSql          = false
-#jpa.generateDDL      = false
+jdbc.driver   = org.h2.Driver
+jdbc.url      = jdbc:h2:../Manager-Repository;DB_CLOSE_DELAY=-1;DATABASE_TO_UPPER=false;
+jdbc.username = apromore
+jdbc.password = MAcri
 
+jpa.database         = H2
+jpa.databasePlatform = org.eclipse.persistence.platform.database.H2Platform
+jpa.showSql          = false
+jpa.generateDDL      = false
 
+# Liquibase User H2
+liquibase.jdbc.username = apromore
+liquibase.jdbc.password = MAcri
 
 # Database and JPA (MySQL)
 
-jdbc.driver   = com.mysql.jdbc.Driver
-jdbc.url      = jdbc:mysql://localhost:3306/apromore?createDatabaseIfNotExist=true&autoReconnect=true&allowMultiQueries=true&rewriteBatchedStatements=true&characterEncoding=utf-8
-jdbc.username = apromore
-jdbc.password = MAcri
+#jdbc.driver   = com.mysql.jdbc.Driver
+#jdbc.url      = jdbc:mysql://localhost:3306/apromore?createDatabaseIfNotExist=true&autoReconnect=true&allowMultiQueries=true&rewriteBatchedStatements=true&characterEncoding=utf-8
+#jdbc.username = root
+#jdbc.password = MAcri
+#
+#jpa.database         = MYSQL
+#jpa.databasePlatform = org.eclipse.persistence.platform.database.MySQLPlatform
+#jpa.showSql          = false
+#jpa.generateDDL      = false
 
-jpa.database         = MYSQL
-jpa.databasePlatform = org.eclipse.persistence.platform.database.MySQLPlatform
-jpa.showSql          = false
-jpa.generateDDL      = false
+## Liquibase User Mysql
+#liquibase.jdbc.username = db_user
+#liquibase.jdbc.password = 7fHJV41fpJ
 
```

Comment:
'7fHJV41fpJ': is it an encrypted value of the password or is it the password? If it is the password, then it should be encrypted as it can be seen easily in this file.

Reply:
Its just a random passowrd. encryption is a different logic which we can consider later. Its not in this scope

Reply:
Ok if it's a low security risk for showing password of an "All privileges" user in a text file. Any judgement?

Reply:
The use is excactly the same as apromore user at this time.
All previleges is only for apromore db, and as per the mysql5.7 doc All privilege is DML and DDL and not USER management.

I Agree this is not the right way, but thats another scope to consider not only for liquibase but also for apromore user.

Reply:
https://dev.mysql.com/doc/refman/5.7/en/grant.html (Table 13.8, the "Level" information). There's a risk of losing data due to password is revealed in plain text file for 'apromore' and 'liquibase_user', at the same risk level for both users. This risk should be addressed in the future if possible.